### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mackoj**
Add some documentation around options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,27 @@ swaggen generate http://myapi.com/spec --template Templates/Swift  --destination
 
 For the Swift template, a handy option is `name`, which changes the name of the generated framework from the default of `API`. This can be set in the template or by passing in `--option name:MyCoolAPI`.
 
+
+### Option
```
Comment (yonaskolb) [inline README.md]: Can we call this `Swift Template Options`

File: README.md [hunk 2]
```diff
@@ -104,6 +104,27 @@ swaggen generate http://myapi.com/spec --template Templates/Swift  --destination
...
+baseURL | change baseURL in APIClient | `String` | null
+fixedWidthIntegers | whether to use types like Int32 and Int64 | `Bool` | false
+homepage | change homepage in podspec  | `String` | https://github.com/yonaskolb/SwagGen
+modelPrefix | change model by adding a prefix and model file name | `String` | null
+modelSuffix | change model by adding a suffix and model file name | `String` | null
+mutableModels | whether model properties are mutable | `Bool` | true
+name | name of the API | `String` | API
+safeArrayDecoding | filter out invalid items in array instead of throwing | `Bool` | false
+safeOptionalDecoding | set invalid optionals to nil instead of throwing | `Bool` | false
+tagPrefix | ??? | `String` | null
```
Comment (yonaskolb) [inline README.md]: `tagPrefix` is: Prefix for all tags.

eg. if a `login` operation has a tag of `Authorization`
`tagPrefix` of `MyName` would be `API.MyNameAuthorization.Login`

Same goes for `tagSuffix`

File: README.md [hunk 3]
```diff
@@ -104,6 +104,27 @@ swaggen generate http://myapi.com/spec --template Templates/Swift  --destination
...
 For the Swift template, a handy option is `name`, which changes the name of the generated framework from the default of `API`. This can be set in the template or by passing in `--option name:MyCoolAPI`.
 
+
+### Option
+
+List of all available options
+
+name | action | expected values | default value
+--- | --- | --- | ---
+authors | change authors in podspec | `String` | Yonas Kolb
```
Comment (yonaskolb) [inline README.md]: Maybe we can leave out the `change...` from all of these. eg. `authors in podspec`

File: README.md (hunk 1 above)
Comment (mackoj) [reply README.md]: Ok

File: README.md (hunk 2 above)
Comment (mackoj) [reply README.md]: Ok I will fix

File: README.md (hunk 3 above)
Comment (mackoj) [reply README.md]: ok

**Pull request merged**